Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/Xamarin.Forms.GoogleMaps.Android.csproj

```diff
@@ -8,7 +8,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Xamarin.Forms.GoogleMaps.Android</RootNamespace>
     <AssemblyName>Xamarin.Forms.GoogleMaps.Android</AssemblyName>
-    <TargetFrameworkVersion>v6.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v7.0</TargetFrameworkVersion>

```

Comment:
Don't include this file in merge, it's settings only for my working project, based on API V25. This issue was critical for me, because my work project has been stopped, when your plugin isn't working.